Dockerfile.compute-node

```diff
@@ -753,7 +753,7 @@ FROM scratch AS postgres-extensions
 # COPY --from=postgres-cleanup-layer /usr/local/pgsql/lib             /usr/local/pgsql/lib
 # Custom extensions
 COPY --from=pg-anon-pg-build /extensions/anon/lib/ /extensions/anon/lib
-COPY --from=pg-anon-pg-build /extensions/anon/share/extension /extensions/anon/share/extension
+RUN --from=pg-anon-pg-build tar -zcvf /extensions/anon/share/extension/anon.tar.gz /extensions/anon/share/extension
```

Comment:
This probably doesn't do what we want.
We need to create an archive in `pg-anon-pg-build` step and COPY it to the extension directory here.

And don't we want to add `libs` to the archive as well, or am I missing something?

Reply:
ah yes, I've hopefully fixed this issue.

Reply:
let me know if it's still broken